watcher_fen.go

```diff
@@ -97,9 +98,9 @@ func (f *fen) Init() (err error) {
 }
 
 func fi2fo(fi os.FileInfo, p string) FileObj {
-	st, ok := fi.Sys().(*syscall.Stat_t)
+	st, ok := fi.Sys().(*unix.Stat_t)
```

Comment:
`fi` is a regular `os.FileInfo` from `os.Stat`. As such `fi.Sys()` returns standard `*syscall.Stat_t`.